### PR TITLE
feat(report-package) Added README for report package

### PR DESCRIPTION
**Pull request by markreay**
#### Pull request checklist

- [n/a] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [n/a] Added/updated relevant unit test(s) (and ran `yarn test`)
- [n/a] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/reports/package/root/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { reporterFactory } from "accessibility-insights-report";
+
+test('my accessibility test', async () => {
+    // This example uses axe-puppeteer, but you can use any axe-based library
+    // that outputs axe scan results in the default axe output format
+    const testPage: Puppeteer.Page = /* ... set up your test page ... */;
+    const axeResults: Axe.AxeResults = await new AxePuppeteer(testPage).analyze();
+
+    // Perform the conversion
+    const html = reporter.fromAxeResult(axeResults).asHTML();
```
Comment (devaradhanm) [inline src/reports/package/root/README.md]: should this be reporterFactory ?

File: src/reports/package/root/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Publish [axe-core](https://github.com/dequelabs/axe-core) accessibility scan results as 
+self-contained HTML files in the same format as
+[accessibility-insights-web](https://github.com/microsoft/accessibility-insights-web).
+
+## Usage
+
+Before using accessibility-insights-report, you will need to run an [axe](https://github.com/dequelabs/axe-core) accessibility scan to produce some axe results to convert. Typically, you would do this by using an axe integration library for your favorite browser automation tool ([axe-puppeteer](https://github.com/dequelabs/axe-puppeteer), [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs), [cypress-axe](https://github.com/avanslaars/cypress-axe)).
+
+axe-sarif-converter exports a factory that can be used to create a report object and output its results as HTML.
```
Comment (smoralesd) [inline src/reports/package/root/README.md]: should it be `accessibility-insights-report` instead of `axe-sarif-converter`?

Comment (markreay) [reply src/reports/package/root/README.md]: Yes indeed. Thanks.

Fixed.

File: src/reports/package/root/README.md (hunk 1 above)
Comment (markreay) [reply src/reports/package/root/README.md]: Actually there was a missing line. Good catch. Thanks.

Fixed.

**Pull request merged**